include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -50,6 +50,15 @@ class MGConstrainedDoFs : public Subscriptor
    * edges of a refinement level are treated corrected but leaves degrees of
    * freedom at the boundary of the domain untouched assuming that no
    * Dirichlet boundary conditions for them exist.
+   *
+   * Furthermore, this call sets up a ConstraintMatrix on each level that
```

Comment:
do we want to build those regardless of their usage? in other words, can this be potentially a time/memory -consuming operation? If so, maybe add a flag to specify whether or not `level_constraints` should be built.